docs/en/development/build-cross-osx.md

```diff
@@ -60,3 +70,46 @@ ninja
 ```
 
 The resulting binary will have a Mach-O executable format and can’t be run on Linux.
+
+# ARM (aarch64) macOS:
```

Comment:
Thanks!

There is quite a bit of copypasta here (which will make future maintenance more difficult). Could we maybe only keep the description for x86 and add comments à la "To build for ARM, replace `x86` by `aarch64`".

Reply:
sure